src/theme/index.ts

```diff
@@ -1,5 +1,4 @@
 import { extendTheme } from "@chakra-ui/react"
-import { mode } from "@chakra-ui/theme-tools"
 import { defaultTheme } from "@threshold-network/components"
```

Comment:
Non-blocking:

This should be addressedi na separate PR, but ideally we should not use `@threshold-network/components` in this repo as teh designs are not related to threshold dashboard at all 😉 Just pointing that out to not forget about it and I want to know if you agree.

Reply:
Correct. I agree. We just have to consider whether it's worth the effort if we want to start a completely new project anyway. I leave the decision up to you 🙇🏼

Reply:
Probably not. We can always return to this issue if it turn out that we will continue from this codebase.